### PR TITLE
Date homepage display 

### LINKED ISSUE #228: Date format on home
**User Story**
As a .... , I want to .... so that ....

image

**Acceptance Criteria**
1. Change Format from June 16 to 16.06.2021
2. Remove Live poll - 6/14/2021

### PR DESCRIPTION
**Pull request by StevenMalaihollo**
Closes #228
also removed the homepage description text

### CODE REVIEW

File: client/src/views/Home.vue [hunk 1]
```diff
@@ -3,10 +3,7 @@
     <div class="flex justify-center items-end w-full background-home h-64 sm:h-128">
         <div class="banner-text flex items-center">
             <div class="max-w-md text-center text-blue-900">
-                <h1 class="font-bold mb-3">{{$t('titles.home')}}</h1>
-                <p class="hidden sm:block mb-8 pb-3">
```
Comment (JakubC-projects) [inline client/src/views/Home.vue:7]: why do we exactly remove that? In figma it is still there

File: client/src/views/Lobby.vue [hunk 2]
```diff
@@ -1,10 +1,9 @@
 <template>
     <section>
         <div class="max-w-screen-md mx-auto px-4 md:px-6">
-            <div class="flex flex-col justify-center" :style="`height: 15vh`">
+            <div class="flex flex-col justify-center" :style="`height: 10vh`">
                 <div class="text-white">
                     <h3 class="font-bold">{{pollingEvent.title}}</h3>
-                    <label>{{'Live poll'}} - {{new Date(pollingEvent.startDateTime).toLocaleDateString()}}</label>
```
Comment (JakubC-projects) [inline client/src/views/Lobby.vue:7]: Weren't we supposed to remove only the "Live poll" label and leave the date?

File: client/src/views/Home.vue (hunk 1 above)
Comment (StevenMalaihollo) [reply client/src/views/Home.vue:7]: Amelie requested this

Comment (JakubC-projects) [reply client/src/views/Home.vue:7]: ok

**Comment by StevenMalaihollo** [comment]
> Weren't we supposed to remove only the "Live poll" label and leave the date? 

In the design it is removed totally

**Review by JakubC-projects — APPROVED** [review]

**Pull request merged**